static/js/panels.js

```diff
@@ -116,6 +124,12 @@ exports.initialize = function () {
         }
         resize_app();
     });
+
+    // Configure keyup handlers.
+    $(".request-desktop-notifications").on("keyup", handle_keyup);
+    $(".reject-notifications").on("keyup", handle_keyup);
+    $(".accept-bankruptcy").on("keyup", handle_keyup);
+    $("#panels").on("keyup", ".alert .exit", handle_keyup);
```

Comment:
These four selectors could probably be replaced with just `$("#panels *[role=button]")`. Since the `handle_keyup` function is very simple, I think it would be better moved here as an anonymous lambda function.

Reply:
Yeah, I think that's better.